src/main/java/browser/BrowserSize.java

```diff
@@ -0,0 +1,7 @@
+package browser;
+
+public enum BrowserSize {
+    DESKTOP,
+    SMALL_DESKTOP,
```

Comment:
small desktop - it's not clear for size naming, choose smth else (laptop for ex)